iOS/Source/MainView.swift

```diff
@@ -201,8 +201,8 @@ class MainView: UIView {
 
             UIView.animate(withDuration: 0.4) {
                 self.backgroundColor = backgroundColor
-                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
                 self.sunView.update(for: location)
+                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
```

Comment:
Yeah we should! I had a weird bug and thought it might had something to do with this. It is a factor of cunfusion! So needs a rewrite.